blocks/quilted-image-block/README.md

```diff
@@ -0,0 +1,28 @@
+# @wpmedia/quilted-image-block
+
+The Quilted Image block is a collection of three images, each with a text and button overlay. In a mobile viewport all three images will display above one another. On a larger viewport, the top or bottom image will become full width with the other two images stacked horizontally adjacent to it.
+
+## Props
+
+| **Prop**                | **Required** | **Type**       | **Default Value** | **Description**                                                                |
```

Comment:
Don't think we need to document these here as blocks are not to be consumed by anything but PageBuilder

Reply:
Got it, thanks!